megaparsec.cabal

```diff
@@ -68,23 +68,22 @@ library
         transformers >=0.4 && <0.7
 
     if flag(dev)
-        ghc-options: -Wall -Werror -Wpartial-fields -Wunused-packages
+        ghc-options:
+            -Wall -Werror -Wredundant-constraints -Wpartial-fields
```

Comment:
I vaguely remember that there used to be a good reason not to use `-Wredundant-constraints`, but if CI passes we can totally bring it back.

Reply:
It had false positives in 9.6.2, but this was fixed in 9.6.3.